vmware-event-router/hack/create_k8s_config.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+# Copyright 2019 VMware, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-2
+
+set -euo pipefail
+
+black='\E[30;40m'
+red='\E[31;40m'
+green='\E[32;40m'
+magenta='\E[35;40m'
+cyan='\E[36;40m'
+reset='\033[00m'
+
+cecho () {
+    local default_msg="No message passed."
+
+    message=${1:-$default_msg}
+    color=${2:-$black}
+
+    echo -e "$color"
+    echo -e "$message"
+    tput sgr0
+
+    return
+}
+
+cecho "Enter the following values for deployment:" $cyan
+
+echo -e "${magenta}"
+read -p "vCenter Server FQDN: " VCENTER_SERVER
+if [[ -z "${VCENTER_SERVER}" ]]; then
+    cecho "Please start over. No input entered." $red
+    exit 1
+fi
+
+echo -e "${magenta}"
+read -p "vCenter Server Username: " VCENTER_USERNAME
+if [[ -z "${VCENTER_USERNAME}" ]]; then
+    cecho "Please start over. No input entered." $red
+    exit 1
+fi
+
+echo -e "${magenta}"
+echo -n "vCenter Server Password: "
+read -s VCENTER_PASSWORD
+if [[ -z "${VCENTER_PASSWORD}" ]]; then
+    cecho "Please start over. No input entered." $red
+    exit 1
+fi
+echo ""
+
+echo -e "${magenta}"
+read -p "Deploy OpenFaaS: [y|n] " DEPLOY_OPENFAAS
+if [[ -z "${DEPLOY_OPENFAAS}" ]]; then
+    cecho "Please start over. No input entered." $red
+    exit 1
+fi
+
+if [ ${DEPLOY_OPENFAAS} == "y" ]; then
+    echo -e "${magenta}"
+    echo -n "OpenFaaS Admin Password: "
+    read -s OPENFAAS_PASSWORD
+    if [[ -z "${OPENFAAS_PASSWORD}" ]]; then
+        cecho "Please start over. No input entered." $red
+        exit 1
+    fi
+fi
+echo ""
+
+cecho "Please confirm the following settings are correct:\n" $cyan
+echo -e "\tVCENTER_SERVER=${VCENTER_SERVER}"
+echo -e "\tVCENTER_USERNAME=${VCENTER_USERNAME}"
+echo -e "\tVCENTER_PASSWORD=${VCENTER_PASSWORD}"
+echo -e "\tDEPLOY_OPENFAAS=${DEPLOY_OPENFAAS}"
+if [ ${DEPLOY_OPENFAAS} == "y" ]; then
+    echo -e "\tOPENFAAS_PASSWORD=${OPENFAAS_PASSWORD}"
+fi
+
+echo -e "${cyan}"
+read -p "Do you want to proceed with the VEBA K8s deployment [y]: " answer
+case $answer in
+    [Yy]* ) cecho "Starting Deployment ..." $green;;
+    [Nn]* ) cecho "Exiting ..." $red; exit;;
+    * ) cecho "Exiting ..." $red; exit;;
+esac
+
+cecho "Checking for K8s namespace creation permission ..." $green
+kubectl auth can-i create ns -q > /dev/null 2>&1
```

Comment:
This is not needed `> /dev/null 2>&1`

```suggestion
kubectl auth can-i create ns -q
```